mkt/api/tests/test_handlers.py

```diff
@@ -678,7 +678,18 @@ def test_banner_message(self):
         eq_(res.status_code, 200)
         data = json.loads(res.content)
         eq_(data['banner_message'], unicode(geodata.banner_message))
-        eq_(data['banner_regions'], [mkt.regions.ARG.slug, mkt.regions.BRA.slug])
+        eq_(data['banner_regions'],
+            [mkt.regions.ARG.slug, mkt.regions.BRA.slug])
+
+    def test_region_v1(self):
+        res = self.client.get(self.get_url)
+        eq_(res.status_code, 200)
+        eq_(res.json['regions'][0]['mcc'], '722')
+
+    def test_region_v2(self):
+        res = self.client.get(self.get_url.replace('v1', 'v2'))
```

Comment:
This seems fragile. The v1 test too. Not sure what to do about it though. Could add an assert that the right version is in the URL I guess.